src/components/Player.js

```diff
@@ -15,21 +15,18 @@ export default function Player(props) {
   function onError() {
     const internalPlayer = props.playerRef.current?.getInternalPlayer()
     const videoUrl = internalPlayer.getVideoUrl()
-    if (!videoUrl) return
-
     const videoId = getVideoId(videoUrl)
```

Comment:
Will this break `getVideoId`? What does that return if `videoUrl` is `undefined` ?

Reply:
error message looks like 
```
BROKEN VIDEO: null, index: 0, videoUrl: https://www.youtube.com/watch, video: {"title":"Tandava - Mangalam [Chillums at Dawn Remix]","video_id":"u4LFC7Q9Ajc","playlist_id":"PL96VlwoPgKeyYyH44K_6M423JbWdxp49G","playlist_owner":"Robert S"}
```